hal/src/thumbv7em/clock.rs

```diff
@@ -80,9 +80,9 @@ pub struct GClock {
     freq: Hertz,
 }
 
-impl Into<Hertz> for GClock {
-    fn into(self) -> Hertz {
-        self.freq
+impl From<GClock> for Hertz {
+    fn from(item: GClock) -> Hertz {
+        item.freq
```

Comment:
I'm not sure if we actually want to implement `From` here. Doesn't that mean we can go both ways? `Hertz` -> `GClock` would be weird haha

Reply:
Haha yeah, you're right. I just blindly trusted clippy on that one. I'll just allow the lint instead.